src/util/checkoutUtils.js

```diff
@@ -85,6 +85,10 @@ export function formatTransactionData(
 		depositTotal: totals.creditAmountNeeded || 0,
 		paymentType,
 		isFTD: false,
+		optimizely: {
+			loanTotal,
+			donationTotal
```

Comment:
Does this value match what you want to send? I think it can come back sometimes looking like `4.5` or just `25`... What type of values did you see?